### ISSUE TITLE
Needs more documentation for self-hosted fonts

### ISSUE THREAD

**Issue by mdizo** [hupe1980/gatsby-plugin-webfonts#5]

**Comment by hupe1980** [comment]
@mdizo The plugin currently only supports self-hosted google fonts. The plugin is designed so that other fonts can be supported. However, due to lack of time and specific requirements, I was not yet able to do this. A PR is welcome 😉

**Comment by ivmarcos** [comment]
@hupe1980 not sure it's something you guys could do, but it would be good to update the information at https://www.gatsbyjs.com/plugins/gatsby-plugin-webfonts, at least until it is released, since it's mentioned there it supports self-hosting.

**Comment by Zauberfisch** [comment]
agreed. please update docs to reflect that hosting your own font files is not supported

**Comment by hupe1980** [comment]
@visualfanatic is currently creating this feature. Here #10 can see the progress.

**Comment by Lane** [comment]
@hupe1980 @visualfanatic would be happy to help getting this feature merged in, is there anything I can do? (documentation, tests, etc.)

**Comment by hupe1980** [comment]
@visualfanatic Any updates?

**Comment by jfgilmore** [comment]
Is this resolved by #50?
